packages/corvid-cli/src/messages/messages_en.json

```diff
@@ -4,6 +4,8 @@
     "Cli_Unsupported_Node_Version": "Unsupported Node.js version, please use",
     "Clone_Getting_Site_Data": "Getting site information",
     "Clone_No_MetasiteId_Error": "Could not get site information from <%= url %>",
+    "Clone_Not_Owner_Error": "User doesn't have perimissions for site <%= url %>",
```

Comment:
you should make sure @wixysam approves the final texts

Reply:
I think we should change to:

+ You don't have permissions for the site <%= url %>.
+ The site URL <%= url%> is invalid.

Reply:
Fixed